controllers/villains.js

```diff
@@ -0,0 +1,41 @@
+const { villains } = require('../models/index')
+
+const getAllVillains = async (request, response) => {
+  const listOfVillains = await villains.findAll()
+
+  return response.send(listOfVillains)
+}
+
+const getVillainBySlug = async (request, response) => {
+  const slugSearch = request.params.slug
+
+  const foundSlug = await villains.findOne({
+    where: { slug: slugSearch }
```

Comment:
you can use the attribute property which will only display certain attributes on the page.

const foundSlug = await villains.findOne({ where: { slug: slugSearch }, attributes: ['name', 'movie', 'slug'], })